src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -784,7 +784,7 @@ public function unsignedDecimal($column, $total = 8, $places = 2)
      */
     public function boolean($column)
     {
-        return $this->addColumn('boolean', $column);
+        return $this->addColumn('boolean', $column, ['unsigned' => true]);
```

Comment:
IMHO this is the wrong please for a database specific change, see \Illuminate\Database\Schema\Grammars\MySqlGrammar::typeBoolean for a suggestion.